src/libraries/Common/tests/System/Security/Cryptography/MLKemTests.Keys.cs

```diff
@@ -221,6 +239,58 @@ public static void ExportDecapsulationKey_OnlyHasEncapsulationKey()
                 new byte[vector.Algorithm.DecapsulationKeySizeInBytes]));
         }
 
+        [ConditionalTheory(typeof(MLKem), nameof(MLKem.IsSupported))]
+        [InlineData(true)]
+        [InlineData(false)]
+        public static void SubjectPublicKeyInfo_MLKem512_Ietf(bool tryExport)
+        {
+            using MLKem kem = MLKem.ImportPrivateSeed(MLKemAlgorithm.MLKem512, IncrementalSeed);
+            AssertSubjectPublicKeyInfo(kem, tryExport, MLKemTestData.IetfMlKem512Spki);
+        }
+
+        [ConditionalTheory(typeof(MLKem), nameof(MLKem.IsSupported))]
+        [InlineData(true)]
+        [InlineData(false)]
+        public static void SubjectPublicKeyInfo_MLKem768_Ietf(bool tryExport)
+        {
+            using MLKem kem = MLKem.ImportPrivateSeed(MLKemAlgorithm.MLKem768, IncrementalSeed);
+            AssertSubjectPublicKeyInfo(kem, tryExport, MLKemTestData.IetfMlKem768Spki);
+        }
+
+        [ConditionalTheory(typeof(MLKem), nameof(MLKem.IsSupported))]
+        [InlineData(true)]
+        [InlineData(false)]
+        public static void SubjectPublicKeyInfo_MLKem1024_Ietf(bool tryExport)
+        {
+            using MLKem kem = MLKem.ImportPrivateSeed(MLKemAlgorithm.MLKem1024, IncrementalSeed);
+            AssertSubjectPublicKeyInfo(kem, tryExport, MLKemTestData.IetfMlKem1024Spki);
+        }
+
+        private static void AssertSubjectPublicKeyInfo(MLKem kem, bool tryExport, ReadOnlySpan<byte> expectedSpki)
```

Comment:
When reading this at the caller, I read "tryExport" as "should I try exporting it after importing it?", which seemed weird.  Maybe "useTryExport"/"useTry"/"viaTry"/etc?